specs/browser-extension/storage.md

```diff
@@ -2,12 +2,17 @@
 
 ## Requirements
 
-- The user's generated mnemonic should be stored in the browser from the extension (encrypted with the password provided during Account Creation)
-- Any derived keys will also need to be stored and encrypted in a similar fashion
+The browser extension incorporates the following storage requirements:
+
+- The user's generated mnemonic is in the browser from the extension (encrypted with the password provided during Account Creation) using an IndexedDB key/value store
```

Comment:
I think You missed the "stored" :)
The user's generated mnemonic is **stored** in...

Reply:
Good catch, thanks!